examples/python-docker/Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[requires]
+python_version = "3.7"
+
+[packages]
+constructs="3.0.4"
+cdktf="0.0.12"
```

Comment:
I would use the latest version of cdktf package.

Reply:
0.0.12 is the latest version for cdktf as per the Pypi.

Reply:
@anubhavmishra  meant `cdktf@next`, however we [don't release](https://pypi.org/project/cdktf/#history) pre releases for Python at the moment. I'm not even sure, if there's something comparable to `cdktf@next` in pypi.

However, I think for the examples we should stick to the monorepo references [as here](https://github.com/hashicorp/terraform-cdk/blob/b8aeded4cb58a7f367195d5ed8a4632588f43eca/examples/python-aws/Pipfile)

```
[[source]]
name = "pypi"
url = "https://pypi.org/simple"
verify_ssl = true

[requires]
python_version = "3.7"

[packages]
cdktf = {path = "./../../dist/python/cdktf-0.0.0-py3-none-any.whl"}
```

Reply:
@skorfmann Pipfile updated to use the monorepo.

Reply:
Cool, thanks 👍